packages/core-components/src/layout/Sidebar/SidebarSubmenuItem.tsx

```diff
@@ -149,7 +150,7 @@ export const SidebarSubmenuItem = (props: SidebarSubmenuItemProps) => {
   const currentLocation = useLocation();
   let isActive = isLocationMatch(currentLocation, toLocation, exact);
 
-  const [showDropDown, setShowDropDown] = useState(false);
+  const [showDropDown, setShowDropDown] = useState(props.expanded ?? false);
```

Comment:
mmm I wonder if something like `initialShowDropDown` would be more appropriate since the new prop sets only the initial state

Reply:
Makes sense to me! Updating now.